roles/download/defaults/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
 local_release_dir: /tmp/releases
+download_cache_dir: "temp/kubespray_cache"
```

Comment:
what is the absolute path out of curiosity ? woudn't make sense to have it in `/tmp/kubespray_cache` and keep it consistent with `local_release_dir` ?

Reply:
The absolute path would be the root of the repository, where cluster.yml resides, appended with temp/kubespray_cache. I have no particular opinion on what the path should default to, so I've changed it to /tmp/kubespray_cache, as per your suggestion. The reason I used a relative path, by the way, is so that the files would be cached to disk, and not on /tmp, as that is tmpfs on many systems, hogging precious RAM. But I'll just symlink it to some non-ephemeral storage on my machine and it'll be all right.